keps/sig-apps/20200529-delete-priority-annotations.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Add delete priority annotation for ReplicaSet
+authors:
+  - "@drbugfinder-work"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+reviewers:
+  - "@cheftako"
+  - "@deads2k"
+  - TBD
+approvers:
+  - "@cheftako"
+  - "@deads2k"
```

Comment:
Given that the owning SIG is Apps, please pick reviewers/approvers from the SIG: https://github.com/kubernetes/kubernetes/blob/b0c1fd19fcb6cc508bb6aa461594eac9e456960a/OWNERS_ALIASES#L268-L285